ee/clickhouse/queries/clickhouse_retention.py

```diff
@@ -235,12 +239,11 @@ def _retrieve_people_in_period(self, filter: RetentionFilter, team: Team):
             },
         )
         people_dict = {}
-        all_people = sync_execute(
-            "SELECT * FROM person WHERE id IN %(uuids)s", {"uuids": [val[0] for val in query_result]}
```

Comment:
Wonder if https://github.com/PostHog/posthog/issues/3085 would have caught this :thinking:

Reply:
probably? although I removed this query here not because of performance but because it's more consistent to pull the person from postgres so the frontend handling will be exactly the same